org.freedesktop.tuhi.json

```diff
@@ -106,8 +107,9 @@
             "buildsystem": "simple",
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://github.com/mozman/svgwrite.git"
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/a6/e1/8d592fc801e1dc2958fe0c84c733ed729d4020daa1826c58978f9d601bb4/svgwrite-1.1.12.zip#md5=05780a4a8ba33c16842faf37818d670e",
+                    "sha512": "cafce1a39c932149cd9d531c7304e8568fde9bc5abd09b548a778cc7f7f4d74327da03fb01279a3a19dd8396487c8fba974d50c6dd645be3942a4ba6d76646df"
```

Comment:
I think using the github package is better here than the pypi cache, removes one layer of indirection:
https://github.com/mozman/svgwrite/archive/v1.1.12.tar.gz

Reply:
Don't we have issues with dates when pulling these tags from github?

Reply:
IIRC things attached to a proper release don't change (libwacom e.g. adds the md5sums to those assets too) so we should be able to use those for svgwrite at least - that link above.